### PR TITLE
Makes areas invisible unless needed

### PR DESCRIPTION
**Pull request by MrPerson**
This should hopefully improve client performance due to having fewer objects to draw. Might also help with that teleporting crash.

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
Can you explain what this does in greater detail?

**Comment by Razharas** [comment]
I doubt this will have any effect

**Comment by MrPerson** [comment]
I doubt it will have any effect either, but I'm not 100% sure and this took maybe 5 minutes to do.

Instead of having BYOND render an invisible sprite and having the area in all oview() and view() checks, instead we'll simply tell BYOND not to even consider the area for visibility or rendering at all.

File: code/game/area/areas.dm [hunk 1]
```diff
@@ -27,24 +27,24 @@
...
+	master = src
 	uid = ++global_uid
 	related = list(src)
 
 	if(requires_power)
 		luminosity = 0
 	else
-		power_light = 1			//rastaf0
-		power_equip = 1			//rastaf0
-		power_environ = 1		//rastaf0
```
Comment (Anonus) [inline code/game/area/areas.dm]: Ah yes, commenting on vars.

**Comment by Aranclanos** [comment]
can't wait to see everything break

**Comment by tuypo1** [comment]
what aran said

although im not sure what it will end up breaking

**Pull request merged**